packages/grafana-ui/src/components/Forms/Field.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Input } from '../Input/Input';
+import { Field } from './Field';
+import { Select } from '../Select/Select';
+
+describe('Field', () => {
+  it('renders the label', () => {
+    const { getByText } = render(
```

Comment:
Instead of getting queries from render we mostly use `screen`, as recommended approach.